docroot/modules/custom/mass_metatag/src/Service/MassMetatagUtilities.php

```diff
@@ -99,6 +100,32 @@ public function getAllOrgsFromNode(Node $node, bool $parent_only = FALSE, bool $
     return $result;
   }
 
+  /**
+   * Gets Organization names for the passed node.
+   *
+   * @param \Drupal\node\Entity\Node $node
+   *   The node to get Orgs and parent Orgs from.
+   *
+   * @return string[]
+   *   The array of slugified Org names related to this node.
+   */
+  public function getOrgsFromNode(Node $node) {
+    $result = [];
+    if ($node->bundle() === 'org_page') {
+      $result[] = $this->slugify(trim($node->label()));
+    }
+    if ($node->hasField('field_organizations')) {
+      /** @var \Drupal\node\Entity\Node[] $org_pages */
+      $org_pages = $node->field_organizations->referencedEntities();
+      foreach ($org_pages as $org_page) {
+        if ($org_page instanceof OrgPageBundle) {
+          $result[] = $this->slugify(trim($org_page->label()));
+        }
+      }
+    }
+    return array_unique($result);
+  }
```

Comment:
I hope we can do a bit of code reorganization here. The most natural way to use this code is $node->getOrg() and not need a helper. We are already setup to allow this. 

1. Copy and rename this function to \Drupal\mass_content\Entity\Bundle\node\NodeBundle. Change $node to $this.
2. Go a step father and override the method in OrgPageBundle (it lives in the same dir)

```
function getOrg() {
    return \Drupal\mass_metatag\Service\MassMetatagUtilities::slugify(trim($this->label()));
}
Note - add 'static' to function definition of `slugify`.
```

This reorg is just a little more work and results in a lot better developer experience.

Reply:
Thanks. I've refactored the code.